tx/client.go

```diff
@@ -172,6 +117,11 @@ func (ub *QueryParamsBuilder) LoadAsset(load bool) *QueryParamsBuilder {
 	return ub
 }
 
+func (ub *QueryParamsBuilder) Pending(pending bool) *QueryParamsBuilder {
```

Comment:
Shall we always set `pending` to true? I remember there was some discussion around this.

Reply:
Default, `pending` is always true. But we will have an option for the user to set it (true|false)